buildpacks/jvm-function-invoker/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-* Updated `libcnb` and `libherokubuildpack` to `0.9.0`. ([#330](https://github.com/heroku/buildpacks-jvm/pull/330))
+* Upgrade `libcnb` and `libherokubuildpack` to `0.10.0`. ([#363](https://github.com/heroku/buildpacks-jvm/pull/363))
+* Buildpack now requires buildpack API version `0.8`. ([#363](https://github.com/heroku/buildpacks-jvm/pull/363))
```

Comment:
Should this (and the ones in the other changelogs) instead say "implements buildpack API version `0.8`" rather than "requires"? Also, I'm presuming this is mentioned due to the change in minimum `lifecycle` version? If so, perhaps that is what needs to be mentioned as the "requires"?

eg:
"Buildpack now implements buildpack API version `0.8` and so requires lifecycle version `0.14.x` or newer"

Reply:
I like that! I will change the wording here and for the other PRs! Thanks!